particles/Experimental/Person_Msg_Demo/PersonMsg.arcs

```diff
@@ -1,40 +1,25 @@
 // Demo: Person-Msg
 
-schema Person
```

Comment:
I think it's worth keeping these schemas and just adding the refinements to the handle connections.

We'll eventually (probably in or after the curly braces change) support refinements on the schema, there's just no obvious syntax for it right now.